scripts/fortran_tools/parse_fortran.py

```diff
@@ -800,6 +800,9 @@ def parse_fortran_var_decl(line, source, run_env):
             if 'allocatable' in varprops:
                 prop_dict['allocatable'] = 'True'
             # end if
+            if 'pointer' in varprops:
```

Comment:
I realized that we don't have the code in to parse Fortran pointers yet, but for the variable debug checks I wanted to include both allocatable and pointer variables. I can take the pointer stuff out if that's not necessary or applicable at all.

Reply:
Let's take this out for now.